### PR TITLE
Added notes related to security pre-notify list requests

### PR DESCRIPTION
**Pull request by manfre**
Documented the existing policy of not accepting security audit/scanning entities to the pre-notification list. Also added a note to help emphasize that people really need to read the instructions if they want to get approved for the list.

### CODE REVIEW

File: docs/internals/security.txt [hunk 1]
```diff
@@ -191,6 +191,12 @@ groups:
    demonstrated ability to responsibly receive, keep confidential and
    act on these notifications.
 
+.. admonition:: Security audit and scanning entities
+
+    As a policy, we do not add these types of entities to the notification
+    list. We are unable to approve all requests and it is not fair to give
+    some a competitive advantage.
```
Comment (carltongibson) [inline docs/internals/security.txt:198]: I think I'd just have the first sentence:

```suggestion
    As a policy, we do not add these types of entities to the notification
    list.
```

File: docs/internals/security.txt [hunk 2]
```diff
@@ -235,3 +241,9 @@ Please also bear in mind that for any individual or organization,
 receiving security notifications is a privilege granted at the sole
 discretion of the Django development team, and that this privilege can
 be revoked at any time, with or without explanation.
+
+.. admonition:: Provide all required information
+
+    A failure to provide the required information in your initial contact
+    will count against you when making the decision on whether or not to
+    approve your request.
```
Comment (carltongibson) [inline docs/internals/security.txt:249]: Does it actually count negative? (Maybe it does... 🙂)

```suggestion
    We cannot process you request unless you provide all the required information. 
```

I don't know. 🤷

Comment (manfre) [reply docs/internals/security.txt:248]: Based upon direct comments from other team members when deciding and my personal views, I'd say it does hurt their chances of approval. It's a bit of a challenge to trust a person with confidential information after they've shown they don't read/follow instructions. 

With that said, I'm not opposed to toning down the wording.

Comment (carltongibson) [reply docs/internals/security.txt:248]: OK, let's keep it. Thanks!

**Pull request merged**